src/UniversalFunctions.jl

```diff
@@ -491,17 +537,225 @@ function psi(uf::Grachev, ζ, tt::HeatTransport)
         term_2 = _b_h / 2 * log1p(_c_h * ζ + ζ^2)
         return -coeff * log_terms - term_2
     else
+        # Nishizawa2018 Eq. A4 (ζ < 0)
         f_h = f_heat(uf, ζ)
         return 2 * log((1 + f_h) / 2)
     end
 end
 
+
+#####
+##### Cheng
+#####
+
+Base.@kwdef struct ChengParams{FT} <: AbstractUniversalFunctionParameters{FT}
+    Pr_0::FT
+    a_m::FT
+    a_h::FT
+    b_m::FT
+    b_h::FT
+    ζ_a::FT
+    γ::FT
+end
+
+"""
+    Cheng <: AbstractUniversalFunction{FT}
+
+# References
+ - [Cheng2005](@cite)
+
+Equations in reference:
+
+    `ϕ_m`: Eq. 22
+    `ϕ_h`: Eq. 24
+    `ψ_m`: Eq. 21
+    `ψ_h`: Eq. 23
+
+# Cheng qnd Brutsaert (2005) functions are applicable in the 
+# stable b.l. regime (ζ >= 0). Businger (1971) functions
+# are applied in the unstable b.l. (ζ<0) regime by
+# default. 
+
+# Fields
+
+$(DSE.FIELDS)
+"""
+struct Cheng{FT, PS <: ChengParams} <: AbstractUniversalFunction{FT}
+    "Monin-Obhukov Length"
+    L::FT
+    params::PS
+end
+
+struct ChengType <: AbstractUniversalFunctionType end
+Cheng() = ChengType()
+
+# Nishizawa2018 Eq. A7
+f_momentum(uf::Cheng, ζ) = sqrt(sqrt(1 - 15 * ζ))
+
+# Nishizawa2018 Eq. A8
+f_heat(uf::Cheng, ζ) = sqrt(1 - 9 * ζ)
+
+function phi(uf::Cheng, ζ, tt::MomentumTransport)
+    FT = eltype(uf)
+    if ζ > 0
+        # Cheng2005 Eq. 22
+        FT = eltype(uf)
+        _a_m = FT(a_m(uf))
+        _b_m = FT(b_m(uf))
+        num = ζ + (ζ^_b_m) * (1 + ζ^_b_m)^((1 - _b_m) / _b_m)
+        denom = ζ + (1 + ζ^_b_m)^(1 / _b_m)
+        return 1 + _a_m * (num / denom)
+    else
+        # Nishizawa2018 Eq. A1 (L < 0)
+        f_m = f_momentum(uf, ζ)
+        return 1 / f_m
+    end
+end
+
+function phi(uf::Cheng, ζ, tt::HeatTransport)
+    FT = eltype(uf)
+    if ζ > 0
+        # Cheng2005 Eq. 24
+        FT = eltype(uf)
+        _a_h = FT(a_h(uf))
+        _b_h = FT(b_h(uf))
+        num = ζ + (ζ^_b_h) * (1 + ζ^_b_h)^((1 - _b_h) / _b_h)
+        denom = ζ + (1 + ζ^_b_h)^(1 / _b_h)
+        return 1 + _a_h * (num / denom)
```

Comment:
Likewise, Eq (11) from https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2004JD004923. Form of $\phi$ is identical for momentum / scalar variables, parameters are not. LGTM.